src/Tests/Microsoft.NET.Sdk.Razor.Tests/AspNetSdkBaselineTest.cs

```diff
@@ -23,7 +23,7 @@ public class AspNetSdkBaselineTest : AspNetSdkTest
 #if GENERATE_SWA_BASELINES
         public static bool GenerateBaselines = true;
 #else
-        public static bool GenerateBaselines = false;
+        public static bool GenerateBaselines = true;
```

Comment:
```suggestion
        public static bool GenerateBaselines = false;
```

We should add a test for this that says that `Assert.False(GenerateBaselines);`

Reply:
Good catch, I just did.